src/client/game.cpp

```diff
@@ -2858,61 +2858,51 @@ void Game::handleClientEvent_HudChange(ClientEvent *event, CameraOrientation *ca
 
 void Game::handleClientEvent_SetSky(ClientEvent *event, CameraOrientation *cam)
 {
-	sky->setVisible(false);
+	// Handle invalid sky type.
```

Comment:
comment is redundant and/or wrong (logging something is not "handling")

Reply:
Must have kept it when I refractored that function, can remove it since what it describes is pretty obvious.